packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1246,6 +1251,18 @@ function updateSuspenseComponent(
           NoWork,
           null,
         );
+
```

Comment:
Suggested comment:

> Because `primaryChildFragment` is a new fiber that we're inserting as the parent of a new tree, we need to set its `treeBaseDuration`.